object-group-game/GetConfiguration.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Configuration;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace object_group_game
+{
+    public static class GetConfiguration
```

Comment:
The class name should be something like Settings or Config. Also it should not be a static class, but perhaps it would be better if it was a Singleton